namui/src/build/types.rs

```diff
@@ -0,0 +1,25 @@
+use serde::{Deserialize, Serialize};
```

Comment:
I think this pr doesn't use `namui`. would you rollback `namui` part in this pr if i'm right? moving `ErrorMessage` to `namui-cli` and remove `namui` from `namui-cli/Cargo.toml` too.

Reply:
e959fb3
Types moved